doc/release/1.14.1-notes.rst

```diff
@@ -2,38 +2,87 @@
 NumPy 1.14.1 Release Notes
 ==========================
 
-This is a bugfix release for some problems found since 1.14.0. The most
-important fixes are the reversion of the multifield index "view" behavior (see
-compatibility notes), and fixes to the spacing in the str and repr of complex
-values.
+This is a bugfix release for some problems reported following the 1.14.0 release. The major
+problems fixed are the following.
 
-The Python versions supported are 2.7 and 3.4 - 3.6. The Python 3.6 wheels
-available from PIP are built with Python 3.6.2 and should be compatible with
-all previous versions of Python 3.6. It was cythonized with Cython 0.26.1,
-which should be free of the bugs found in 0.27 while also being compatible with
-Python 3.7-dev.
+* Problems with the new array printing, particularly the printing of
+  complex values,
+* Problems with ``np.einsum`` with the new ``optimized=True`` default. Some
+  fixes for optimization have been applied and ``optimize=False`` is now the
+  default.
+* The sort order in ``np.unique`` when ``axis=<some-number>`` will now always
+  be lexicographic in the subarray elements. Previously there was an
+  optimization that could sort the subarrays as unsigned byte strings.
+* Revert the change of copy to view in multifield indexing.
 
-
-Compatibility notes
-===================
-
-Multifield Indexing of Structured Arrays will still return a copy
------------------------------------------------------------------
-We have reverted the change in 1.14.0 that multi-field indexing of structured
-arrays returns a view instead of a copy, and pushed it back to 1.15. This will
-give us time to implement additional related bugfixes to ease the transition.
-Affected users should read the the 1.14.1 Numpy User Guide for advice on how to
-manage this transition, under "basics/structured arrays/accessing multiple
-fields". A new method `numpy.lib.recfunctions.repack_fields` has been
-introduced to help with this.  All other changes to structured arrays described
```

Comment:
We're choosing to leave this hidden then @ahaldane?

Reply:
I copied the notes from master, probably this was only in the 1.14.x branch.

Reply:
I thought this would be useful info for users, but if it's not part of the standard update-release-note I don't mind skipping it.

Reply:
I've added back part of this to the list of problems fixed.

Reply:
Looks good to me!